core-blocks/pullquote/index.js

```diff
@@ -132,10 +116,10 @@ export const settings = {
 		},
 
 		save( { attributes } ) {
-			const { value, citation, align } = attributes;
+			const { value, citation } = attributes;
```

Comment:
It shouldn't be removed from the deprecated version because it will break it. In addition:
```js
align: {
	type: 'string',
	default: 'none',
},
```
should be added to the list of attributes.

Reply:
Addressed in [this](https://github.com/WordPress/gutenberg/pull/7100/commits/9374ed318ead245d1b3ba974307c1c7a62bc7096) commit. Thank you ❤️